crypto_adapters/t_cose_psa_crypto.c

```diff
@@ -38,6 +38,34 @@
 #include <psa/crypto.h>     /* PSA Crypto Interface to mbed crypto or such */
 
 
+/*
+ * See documentation in t_cose_crypto.h
+ *
+ * This will typically not be referenced and thus not linked,
+ * for deployed code. This is mainly used for test.
+ */
+bool
+t_cose_crypto_is_algorithm_supported(int32_t cose_algorithm_id)
+{
+    static const int32_t supported_algs[] = {
+        COSE_ALGORITHM_SHA_256,
```

Comment:
Am I correct in assuming that this is just a starting template, and each of these should have an appropriate ifdef so that the proper ones are selected?

Reply:
Definitely for more exotic and variable algorithms. Probably not necessary for SHA-256.  It is probably helpful to have them to reflect the configuration of the crypto library through into t_cose.